src/Compilers/Core/Portable/Syntax/InternalSyntax/SyntaxList.WithLotsOfChildren.cs

```diff
@@ -37,12 +31,6 @@ internal WithLotsOfChildren(ObjectReader reader)
                 _childOffsets = CalculateOffsets(this.children);
             }
 
-            internal override void WriteTo(ObjectWriter writer)
```

Comment:
How about `internal WithLotsOfChildren(ObjectReader reader)` above? Can we remove it?

Reply:
looks like i missed those.  thanks!